pom.xml

```diff
@@ -672,6 +630,48 @@
           </execution>
         </executions>
       </plugin>
+            <plugin>
```

Comment:
Fixed indent.

The re-ordering is half of the build fix here. Maven plugins are run in the order they exist in this file and the (now) latest ones depend on actions in the previous ones. Specifically, enforce-files-exists depends on files made in plugins that need to be run first.